crates/bevy_math/src/ops.rs

```diff
@@ -0,0 +1,291 @@
+//! This mod re-exports the correct versions of floating-point operations with
+//! unspecified precision in the standard library depending on whether the `libm`
+//! crate feature is enabled.
+//!
+//! All the functions here are named according to their versions in the standard
+//! library.
+
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(clippy::disallowed_methods)]
+
+// Note: There are some Rust methods with unspecified precision without a `libm`
+// equivalent:
+// - `f32::powi` (integer powers)
+// - `f32::log` (logarithm with specified base)
+// - `f32::abs_sub` (actually unsure if `libm` has this, but don't use it regardless)
+//
+// Additionally, the following nightly API functions are not presently integrated
+// into this, but they would be candidates once standardized:
+// - `f32::gamma`
+// - `f32::ln_gamma`
+
+#[cfg(not(feature = "libm"))]
+mod std_ops {
+    #[inline(always)]
+    pub(crate) fn powf(x: f32, y: f32) -> f32 {
```

Comment:
Just me wondering:
Could this be done with something like:
```
mod std_ops {
    pub use f32::powf; ///etc...
}
```
or would that not work??

Reply:
I'm not sure, but it also fails to unify the naming of the std and libm versions, which differ in many (most?) cases.

Reply:
You can do `pub use libm::sqrtf as sqrt` and so on

Reply:
You could do `pub use libm::thingf as thing`, I think.

Reply:
True! Do you know if that would dodge the lints? I just organized them this way because it's how glam does it.

Reply:
Hmm, not sure. If it breaks the lints, then it's not worth it and should be kept how it is now

Reply:
Hmm; I tried it and it literally just doesn't work, but maybe I'm just being silly. i.e., `pub use f32::powf;` in a module literally just doesn't compile. I dunno how much this is worth fussing over anyway; if these later become public then the whole module thing will probably have to be reorganized regardless.